first_hour_on_vwb/creating_a_data_collection.ipynb

```diff
@@ -0,0 +1,630 @@
+    "        ], layout=wu.vbox_layout)\n",
+    "\n",
+    "    def get_data_collections(self):\n",
+    "        result = subprocess.run([\"terra\",\"workspace\",\"list\",\"--format=JSON\"],capture_output=True,text=True)\n",
+    "        ids_list = wu.list_data_collections(result.stdout)\n",
+    "        return ids_list\n",
+    "\n",
+    "    def publish_version(self,b):\n",
+    "        with self.output:\n",
+    "            # Save values as variables for reuse.\n",
+    "            workspace_id = self.input_workspace_id.value.strip()\n",
+    "            version = self.input_version.value.strip()\n",
+    "            description = self.input_description.value.strip()\n",
+    "            \n",
+    "            # 1) Point cloud env to target data-collection workspace.\n",
+    "            setWorkspaceCommand = f\"terra workspace set --id={workspace_id}\"\n",
+    "            setWorkspaceResult = subprocess.run(setWorkspaceCommand, shell = True, capture_output = True, text = True)\n",
+    "            print(setWorkspaceResult.stderr) if not setWorkspaceResult.stdout else print(setWorkspaceResult.stdout)\n",
+    "\n",
+    "            # 2) Create version folder.            \n",
+    "            if description is not None:\n",
+    "                createVersionCommand = f\"terra folder create --name={version} --description={description} --workspace={workspace_id}\"\n",
+    "            else:\n",
+    "                createVersionCommand = f\"terra folder create --name={version} --workspace={workspace_id}\"\n",
+    "            createVersionResult = subprocess.run(createVersionCommand, capture_output = True, text = True, shell = True)\n",
+    "            print(createVersionResult.stderr) if not createVersionResult.stdout else print(createVersionResult.stdout)\n",
+    "            \n",
+    "            # 3) Get file tree from Terra CLI.\n",
+    "            tree_dictionary = vfu.parse_tree()\n",
+    "\n",
+    "            # 4) Search tree for ID of desired version folder.\n",
+    "            folder_id = vfu.get_folder_id(version,tree_dictionary)\n",
+    "\n",
+    "            # 6) Publish desired folder as a version with today's date.\n",
+    "            today = date.today()\n",
+    "            formatted_date = today.strftime(\"%Y-%m-%d\")\n",
+    "            publishVersionCommand = f\"terra folder set-property --properties=terra-published-date={formatted_date} --id={folder_id}\"\n",
+    "            publishVersionResult = subprocess.run(publishVersionCommand, shell = True, capture_output = True, text = True, check = True)\n",
+    "            print(publishVersionResult.stderr) if not publishVersionResult.stdout else print(publishVersionResult.stdout)\n",
+    "\n",
+    "publish_version_widget = PublishVersionWidget()\n",
+    "display(publish_version_widget.vb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add resources to a data collection version\n",
```

Comment:
IMO it would be helpful here to have them create a resource in _this_ notebook (e.g. just a controlled bucket, or maybe a referenced resource pointing to something public), and not make them go elsewhere to do this step.  Then add a command to move _that_ resource to the version folder.  

This would have 2 benefits:
- don't make them break the flow by going elsewhere
- the important step to get the resource into the right folder can be runnable, not just markdown that their eyes might glaze over

Reply:
I am planning to do this in a subsequent PR -- I have built the tooling to actually move ALL the contents of a given folder to the version folder, but that section would benefit from its own review. Resolving accordingly.